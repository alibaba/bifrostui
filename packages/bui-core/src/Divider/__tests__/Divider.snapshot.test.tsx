import { snapshotTest } from 'testing';

describe('Divider snapshot', () => {
  snapshotTest('Divider');
});
