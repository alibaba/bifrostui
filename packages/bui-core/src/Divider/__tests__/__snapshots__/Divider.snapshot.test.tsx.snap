// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider snapshot Divider demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "fontSize": "16px",
      "height": "40px",
    }
  }
>
  左边
  <div
    className="bui-divider bui-divider-vertical"
    style={
      Object {
        "margin": "0 6px",
      }
    }
  />
  右边
</div>
`;

exports[`Divider snapshot Divider demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "fontSize": "16px",
      "height": "40px",
    }
  }
>
  左边
  <div
    className="bui-divider bui-divider-vertical bui-divider-vertical-dashed"
    style={
      Object {
        "margin": "0 6px",
      }
    }
  />
  右边
</div>
`;

exports[`Divider snapshot Divider demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "fontSize": "16px",
      "height": "40px",
    }
  }
>
  左边
  <div
    className="bui-divider bui-divider-vertical"
    style={
      Object {
        "margin": "0 6px",
      }
    }
  />
  右边
</div>
`;

exports[`Divider snapshot Divider demo snapshot 0 4`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "fontSize": "16px",
    }
  }
>
  上边
  <div
    className="bui-divider bui-divider-horizontal"
    style={Object {}}
  />
  下边
</div>
`;

exports[`Divider snapshot Divider demo snapshot 0 5`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "row",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "fontSize": "16px",
      "height": "40px",
    }
  }
>
  左边
  <div
    className="bui-divider bui-divider-vertical"
    style={
      Object {
        "height": "12px",
        "margin": "0 6px",
      }
    }
  />
  右边
</div>
`;

exports[`Divider snapshot Divider demo snapshot 0 6`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "fontSize": "16px",
    }
  }
>
  上边
  <div
    className="bui-divider bui-divider-horizontal"
    style={
      Object {
        "width": "100px",
      }
    }
  />
  下边
</div>
`;

exports[`Divider snapshot Divider demo snapshot 0 7`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "fontSize": "16px",
    }
  }
>
  上边
  <div
    className="bui-divider bui-divider-horizontal bui-divider-content"
    style={
      Object {
        "margin": "6px 0",
      }
    }
  >
    <span
      className="bui-divider-line"
      style={
        Object {
          "width": "60px",
        }
      }
    />
    <span
      className="bui-divider-text"
    >
      我是分割线
    </span>
    <span
      className="bui-divider-line"
      style={
        Object {
          "width": "60px",
        }
      }
    />
  </div>
  下边
</div>
`;

exports[`Divider snapshot Divider demo snapshot 0 8`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
      "fontSize": "16px",
    }
  }
>
  上边
  <div
    className="bui-divider bui-divider-horizontal"
    style={
      Object {
        "borderBottomWidth": "2px",
      }
    }
  />
  下边
</div>
`;
